core/container/src/main/java/org/wildfly/swarm/internal/MavenArgsParser.java

```diff
@@ -0,0 +1,54 @@
+package org.wildfly.swarm.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.StringTokenizer;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class MavenArgsParser {
```

Comment:
I agree that this is important for our immediate use case, but I would prefer we implement all cases otherwise we only introduce problems down the road as we think something should work but it doesn't